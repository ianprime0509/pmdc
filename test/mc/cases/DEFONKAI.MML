; Output: DEFONKAI.M
; Console: DEFONKAI.OUT

#Title Transposition
#Option /v/c

ABC _{-b}
DE _{+f}

A abc _{+ac} abc
B bcd _{+b c d} bcd
C cde cde
D def _{=f} def
E efg _{+f,g} def
