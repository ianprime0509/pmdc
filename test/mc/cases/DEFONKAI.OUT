 .MML file --> .M file Compiler ver 4.8s
		Programmed by M.Kajihara(KAJA) 2023/09/23

Part A	Length : 144
Part B	Length : 144
Part C	Length : 144
Part D	Length : 144
Part E	Length : 144
Compile Completed.
